src/main/java/com/puppycrawl/tools/checkstyle/filters/SuppressionPatchXpathFilter.java

```diff
@@ -65,6 +70,12 @@
      */
     private String strategy;
 
+    /**
+     * Set has user defined Checks that need modify violation nodes
+     * to their parent abstract nodes to get their child nodes.
+     */
+    private Set<String> abstractNodeChecks;
+
```

Comment:
@romani I add a config here `abstractNodeChecks`

Reply:
I donot see usage of it in xml config, please do.

Reply:
done